README.md

```diff
@@ -1,6 +1,28 @@
 # About Power Flows DMN Model
 Power Flows DMN Model
 
+# DMN model
+Power Flows model has been designed as a JSON file. The file contains information about input and output data. The additional division is sections with headers and decisions.
+
+```json
+{
+   "headers": {
+     "in": [
+       {"name": "age", "type": "integer"},
+       {"name": "colour", "type": "string"}
+     ],
+     "out": [
+       {"name": "allow", "type": "boolean"}
+     ]
+   },
+   "decisions": [
+     {"age": 3, "allow": true},
+     {"age": 8, "colour": "red", "allow": true},
+     {"colour": "green", "allow": true}
+   ]
```

Comment:
Eg. for a such JSON:

{
   "fields": {
     "in": [
       {"name": "age", "type": "integer", "description": "This is something about age"},
       {"name": "colour", "type": "string", "description": "This is something about colour"}
     ],
     "out": [
       {"name": "allow", "type": "boolean", "description": "We expect a decision, if we have access to do it"}
     ]
   },
   "decisions": [
     {"age": 3, "allow": true},
     {"age": 8, "colour": "red", "allow": true},
     {"colour": "green", "allow": true, "description": "Green allows always"}
   ]
}

We get a such YAML:
fields:
  in:
  - name: age
    type: integer
    description: This is something about age
  - name: colour
    type: string
    description: This is something about colour
  out:
  - name: allow
    type: boolean
    description: We expect a decision, if we have access to do it
decisions:
- age: 3
  allow: true
- age: 8
  colour: red
  allow: true
- colour: green
  allow: true
  description: Green allows always

It's harder to compare between particular decisions. However, quicker to build, since " and {} is no needed.

Reply:
And I found a lib for YAML in JS https://github.com/nodeca/js-yaml

Reply:
However... would be better to have a possibility  to name the same for in and out.

Reply:
Here is without the collision, however a little bit more complex DSL:
```yaml
fields:
  in:
  - name: age
    type: integer
    description: This is something about age
  - name: colour
    type: string
    description: This is something about colour
  out:
  - name: allow
    type: boolean
    description: We expect a decision, if we have access to do it
decisions:
  in:
  - age: 3
  out:
  - allow: true
  in:
  - age: 8
    colour: red
  out:
  - allow: true
  in:
  - colour: green
  out:
  - allow: true
    description: Green allows always
```

Reply:
I added additional fields id, name, hit-policy
```yaml
id: sample_decision_id
name: Sample Decision Name
hit-policy: UNIQUE
fields:
  in:
  - name: age
    type: integer
    description: This is something about age
  - name: colour
    type: string
    description: This is something about colour
  out:
  - name: allow
    type: boolean
    description: We expect a decision, if we have access to do it
decisions:
  in:
  - age: 3
  out:
  - allow: true
  in:
  - age: 8
    colour: red
  out:
  - allow: true
  in:
  - colour: green
  out:
  - allow: true
    description: Green allows always
```